### ISSUE TITLE
 Error 400 when updating a google_spanner_instance's config using processing_units

### ISSUE THREAD

**Issue by sserban-us** [hashicorp/terraform-provider-google#10046]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request.
* Please do not leave _+1_ or _me too_ comments, they generate extra noise for issue followers and do not help prioritize the request.
* If you are interested in working on this issue or have submitted a pull request, please leave a comment.
* If an issue is assigned to the `modular-magician` user, it is either in the process of being autogenerated, or is planned to be autogenerated soon. If an issue is assigned to a user, that user is claiming responsibility for the issue. If an issue is assigned to `hashibot`, a community member has claimed the issue already.

### Terraform Version
Terraform v1.0.6
+ provider.google v3.82.0

### Affected Resource(s)
* google_spanner_instance

### Terraform Configuration Files
Current config:
```
resource "google_spanner_instance" "omega_trade" {
  name         = "omegatrade-instance"
  config       = "regional-us-west1"
  processing_units = 1000
  labels       = { env = "test" }
  project      = "searce-blogs-test-project"
}
 ```

New config I want to apply (update `config`):
```
resource "google_spanner_instance" "omega_trade" {
  name         = "omegatrade-instance"
  config       = "regional-us-west1"
  processing_units = 2000
  labels       = { env = "test" }
  project      = "searce-blogs-test-project"
}
```

### Debug Output

https://gist.github.com/sserban-us/ccbd7cb41c03a3cca203beb8afc30c9c

### Panic Output

N/A

### Expected Behavior

The instance should have been resized correctly to 2000 PUs. This works just fine if I use num_nodes instead of processing_units to set the instance size.

### Actual Behavior

Error updating Instance "searce-blogs-test-project/omegatrade-instance-tf-kzcr": googleapi: Error 400: Invalid UpdateInstance request.
 Details:
│ [
│   {
│     "@type": "type.googleapis.com/google.rpc.BadRequest",
│     "fieldViolations": [
│       {
│         "description": "Must specify a non-empty field mask.",
│         "field": "field_mask"
│       }
│     ]
│   }
│ ]
│ 

### Steps to Reproduce

The complete source code is in this repository: https://github.com/cloudspannerecosystem/spanner-terraform-example. Note that because num_nodes and processing_units are mutually exclusive, I'm commenting out everything related to num_nodes and using processing_units instead.

### Important Factoids

No.

### References

* https://github.com/hashicorp/terraform-provider-google/issues/7732

**Comment by angelchang** [comment]
This is a duplicate of https://github.com/hashicorp/terraform-provider-google/issues/9908 which was fixed here GoogleCloudPlatform/magic-modules#5147. Please wait for the next release.

**Comment by slevenick** [comment]
Closing based on @angelchang's comment
